packages/suite/src/components/suite/AppNavigation.tsx

```diff
@@ -373,13 +340,61 @@ export const AppNavigation = ({ items, primaryContent, maxWidth, inView }: Props
                                                             return true;
                                                         },
                                                         label: title,
-                                                        'data-test': item['data-test'],
                                                     };
                                                 }),
                                             },
                                         ]}
                                     />
-                                ) : undefined}
+                                </AppNavigationTooltip>
+                            )}
+                            <SecondaryMenu visible={!condensedSecondaryMenuVisible}>
+                                {itemsSecondaryWithoutExtra.map(item => {
+                                    const { id, title } = item;
+                                    return (
+                                        <AppNavigationTooltip key={id}>
+                                            <StyledButton
+                                                variant={
+                                                    id === 'wallet-coinmarket-buy'
+                                                        ? 'primary'
+                                                        : 'secondary'
+                                                }
+                                                onClick={item.callback}
+                                                {...(item['data-test'] && {
+                                                    'data-test': item['data-test'],
+                                                })}
+                                                isDisabled={isAccountLoading}
+                                            >
+                                                <Text>{title}</Text>
+                                            </StyledButton>
+                                        </AppNavigationTooltip>
+                                    );
+                                })}
+                                {!!itemsSecondaryWithExtra.length && (
+                                    <AppNavigationTooltip>
+                                        <StyledDropdown
+                                            alignMenu="right"
+                                            offset={5}
+                                            isDisabled={isAccountLoading}
```

Comment:
maybe it's weird to disable the dropdown itself and not just the items in the dropdown menu? But not a big deal... 
![image](https://user-images.githubusercontent.com/3729633/205309714-c6228c03-861b-48e7-9079-c4331a64e227.png)